EncountifyAPI/Data/EncountifyAPIContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using EncountifyAPI.Models;
+
+namespace EncountifyAPI.Data
+{
+#pragma warning disable CS1591
```

Comment:
It seems that this disable warning is when comments are missing, so maybe we can add comments